docs/root/intro/arch_overview/http/http_filters.rst

```diff
@@ -25,3 +25,23 @@ themselves within the context of a single request stream. Refer to :ref:`data sh
 between filters <arch_overview_data_sharing_between_filters>` for more details. Envoy already
 includes several HTTP level filters that are documented in this architecture overview as well as
 the :ref:`configuration reference <config_http_filters>`.
+
+.. _arch_overview_http_filters_ordering:
+
+Filter ordering
+---------------
+
+Filter ordering in the :ref:`http_filters field <envoy_v3_api_field_extensions.filters.network.http_connection_manager.v3.HttpConnectionManager.http_filters>`
+matters. If filters are configured in the following order (and assuming all three filters are
+decoder/encoder filters):
+
+.. code-block:: yaml
+
+  http_filters:
+    - A
+    - B
+    - C
+
+The connection manager will invoke decoder filters in the order: ``A``, ``B``, ``C``.
```

Comment:
It's worth mention that there have to be a terminal filter at the end, usually the router.

Reply:
@lizan good point! Updated, lmk what you think